iOS/DuckDuckGo/CreditCardManagement/List/AutofillCreditCardListViewModel.swift

```diff
@@ -182,7 +181,8 @@ final class AutofillCreditCardListViewModel: CreditCardListViewModelProtocol {
                 cardholderName: oldCard.cardholderName,
                 cardSecurityCode: oldCard.cardSecurityCode,
                 expirationMonth: oldCard.expirationMonth,
-                expirationYear: oldCard.expirationYear)
+                expirationYear: oldCard.expirationYear,
+                created: oldCard.created)
```

Comment:
This is to allow cards that have been “undeleted” to appear back in their original position in the card list (which is sorted by created date)